draft-irtf-cfrg-aegis-aead.md

```diff
@@ -346,9 +346,9 @@ Primitives:
 - `a || b`: the concatenation of `a` and `b`.
 - `a mod b`: the remainder of the Euclidean division between `a` as the dividend and `b` as the divisor.
 - `LE64(x)`: the little-endian encoding of unsigned 64-bit integer `x`.
-- `ZeroPad(x, n)`: padding operation. Trailing zeros are appended to `x` until its length is a multiple of `n` bits, with no padding added if `x` is already a multiple of `n` bits (including when `x` is empty).
-- `Truncate(x, n)`: truncation operation. Returns the first `n` bits of `x`.
-- `Split(x, n)`: splitting operation. `x` is split into `n`-bit blocks, ignoring partial blocks.
+- `ZeroPad(x, n)`: trailing zeros are appended to `x` until its length is a multiple of `n` bits. No padding is added if `x` is already a multiple of `n` bits, including when `x` is empty.
+- `Truncate(x, n)`: returns the first `n` bits of `x`.
+- `Split(x, n)`: `x` is split into `n`-bit blocks, ignoring partial blocks.
```

Comment:
I think this is better. However, we could still start each of these functions with `returns`. For example:

> LE64(x): returns the little-endian encoding of unsigned 64-bit integer x.
> ZeroPad(x, n): returns x appended with trailing zeros until its length is a multiple of n bits. No padding is added if x is already a multiple of n bits, including when x is empty.
> Split(x, n): returns x split into n-bit blocks, ignoring partial blocks.

But AESRound(), Repeat(), and CtEq() are fine as is.

What do you think? Or we could get rid of `returns` in all cases.

Reply:
`trailing zeros` could probably be changed to just `zeros` due to the word `appended` now as well.

Reply:
Oh your next commit addresses this a bit.

Reply:
Yes, it now only says `appends`.

Reply:
changed to start with `returns`.